migrations/20250311054626-update-triggers-for-lookup-table.js

```diff
@@ -0,0 +1,72 @@
+'use strict';
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    await queryInterface.sequelize.query(`
+      DROP TRIGGER IF EXISTS delete_file_from_look_up_table_after_file_deleted ON files;
```

Comment:
Remove the DROP, use CREATE or REPLACE, otherwise we risk to fail the second step and leave the production environment without any trigger at all.